hydromt/components/grid.py

```diff
@@ -169,24 +173,24 @@ def read(
         **kwargs : dict
```

Comment:
But read does not need to be a `hydromt_step` since it won't be called from the yaml right? as I understand it, `read` will be called implicitly by the model/components where necessary and thus does not need to be a step